src/figuresClasses.ts

```diff
@@ -1,11 +1,86 @@
-export interface Figure {}
+export enum Shape {
+  triangle = 'triangle',
+  circle = 'circle',
+  rectangle = 'rectangle',
+}
+export interface Figure {
+  shape: Shape;
+  color: string;
+  getArea: () => number;
+}
+
+export class Triangle implements Figure {
+  shape = Shape.triangle;
+
+  // eslint-disable-next-line no-useless-constructor
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    const values: number[] = [a, b, c].sort((x: number, y: number) => x - y);
```

Comment:
The variables `a`, `b`, and `c` should be prefixed with `this.` when used inside the constructor to refer to the instance variables. Otherwise, it will cause a reference error.